content/es/guide/v10/getting-started.md

```diff
@@ -7,7 +7,6 @@ permalink: '/guide/getting-started'
 
 Esta guía lo ayuda a comenzar a desarrollar aplicaciones Preact. Hay 3 formas populares de hacerlo.
 
-##
```

Comment:
I'm not sure this is the correct fix, this doc should (largely) match the [English version](https://github.com/preactjs/preact-www/blob/master/content/en/guide/v10/getting-started.md)

Reply:
If you see spanish translated version preview, can see ## without any text heading, at first all, it's the only issue I see at the momment to read documentation.